### PR TITLE
Add twitter card

### LINKED ISSUE #34: Twitter preview not available
Previews for Twitter are not working

image

I didn't found a plugin for this or code like this in this theme

```html
<meta name="twitter:card" value="summary" />
<meta name="twitter:url" value="" />
<meta name="twitter:title" value="" />
<meta name="twitter:description" value="" />
<meta name="twitter:image" value="" />
<meta name="twitter:site" value="" />
<meta name="twitter:creator" value="" />
```

### PR DESCRIPTION
**Pull request by avdata99**
Fixes #34

This PR add Twitter card to headers.
I didn't manage to test locally but I assume it's ok

### CODE REVIEW

**Comment by gocemitevski** [comment]
@avdata99 please test this locally with WP debugging turned on. I get the following error:

```php
Parse error: syntax error, unexpected end of file in /wp-content/themes/wordpress-theme/header.php on line 186
```

**Comment by avdata99** [comment]
I add some [curly braces here](https://github.com/okfn/wordpress-theme/pull/35/commits/3969b62f9d3ce75d75e25f28d9fc55b074a0446e) and it no longer gives the error @gocemitevski

File: header.php [hunk 1]
```diff
@@ -15,6 +15,37 @@
...
+			if (!$twitter_name) { $twitter_name = 'okfn'; }
+			?>
+			<meta name="twitter:card" value="summary" />
+			<meta name="twitter:url" value="<?php echo $twitter_url; ?>" />
+			<meta name="twitter:title" value="<?php echo $twitter_title; ?>" />
+			<meta name="twitter:description" value="<?php echo $twitter_desc; ?>" />
+			<?php if ($twitter_thumb) { ?>
+			<meta name="twitter:image" value="<?php echo $twitter_thumb; ?>" />
+			<?php } ?>
+			<meta name="twitter:site" value="@okfn" />
```
Comment (gocemitevski) [inline header.php]: @avdata99 I think we need to use `$twitter_name` name for `twitter:site`, instead of having it hard-coded to @okfn.

**Review by gocemitevski — CHANGES_REQUESTED** [review]

File: header.php (hunk 1 above)
Comment (avdata99) [reply header.php]: I've found the `okfnwp_twitter_id` theme option. Maybe it's better. Updated [here](https://github.com/okfn/wordpress-theme/commit/9781662cf4f5676c8e4b721cc59541349eb99a7f) @gocemitevski

**Comment by gocemitevski** [comment]
@avdata99 looks good.

**Pull request merged**